runtime/src/main/java/org/corfudb/runtime/collections/TableMetrics.java

```diff
@@ -0,0 +1,224 @@
+package org.corfudb.runtime.collections;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.MeterRegistry;
+import com.google.gson.JsonObject;
+import org.corfudb.common.metrics.micrometer.MeterRegistryProvider;
+
+import java.util.Optional;
+
+
+/**
+ * List of metrics captured at a CorfuStore Table level
+ *
+ * Created by hisundar on 2020-09-22
+ *
+ */
+public class TableMetrics {
+    private final String tableName;
+    private final Optional<Counter> numPuts;
```

Comment:
do you need "num" prefix in all these field names? Could it be a good idea to get rid of the prefix?

Reply:
done, thanks